cmd/upgrade/operatorroles/cmd.go

```diff
@@ -347,6 +349,11 @@ func createOperatorRole(mode string, reporter *rprtr.Object, awsClient aws.Clien
 		if err != nil {
 			return err
 		}
+		// if an operators upgrade is triggered from the 'upgrade cluster' command,
+		// wait before creating an upgrade policy to avoid the retry loop in CS
+		if isProgrammaticallyCalled {
```

Comment:
Why only programmaticallyCalled?

Reply:
I thought that if a user calls `rosa upgrade operator-roles` from the console, and afterward `rosa upgrade cluster` there is going to be enough time for the roles to reconcile.

Should I remove this `if` statement so we add the delay for all cases?

Reply:
I think so  we cover all scenarios

Reply:
The `if` statement was removed.